tests/integration/targets/ec2_asg_instance_refreshes_info/aliases

```diff
@@ -0,0 +1,2 @@
+cloud/aws
+ec2_asg_instance_refreshes_info
```

Comment:
No need to duplicate the tests here.  You can merge the ec2_asg_instance_refreshes_info and ec2_asg_instance_refresh targets and add ec2_asg_instance_refreshes_info to the aliases file

Reply:
I think I understand what you're saying, and it would of course make sense to combine the two test-"suites" for these related modules into one. However, I'm not 100% sure I did it correctly. Calling each still only runs the tests only for that module. Attempted fix in da9fe1bf6e9571657aece73d57692341fb044926